www/sejd00/task/task.html

```diff
@@ -0,0 +1,55 @@
+    <div class="rgb-color">This text is green-ish using a rgb color</div>
+    <div class="hex-color-bold">This text is blue-ish using a hex color and the text is bold</div>
+    <div class="font-s">The size of this text is 18px</div>
+    <div class="font-s-italic">The size of this text is 24px and the text is italic</div>
+    <div class="bg-color">This text is white and the background color is brown-ish</div>
+    <p class="par-1-centered">This paragraph's content is centered and the line height is 32px. Lorem ipsum dolor sit
+        amet consectetur
+        adipisicing elit.
+        Earum excepturi optio consectetur molestias deleniti, iure voluptas reiciendis, dolor iste voluptatem iusto
+        aliquid unde, quam commodi a doloribus impedit numquam assumenda.</p>
+    <p class="par-2-justify">This paragraph's content is justified and the text is capitalized everywhere (use
+        text-transform). Lorem ipsum
+        dolor, sit amet consectetur adipisicing elit. Expedita ipsa perferendis quod quia obcaecati illum enim saepe,
+        maxime, ratione iure iusto. Quae delectus voluptatibus, repellendus veniam facere molestias dolor possimus!</p>
+    <div class="box-01">This box is 400px in width and 300px in height total. It has a solid 1px purple border and
+        the padding is 30px</div>
+    <div class="box-02">This box is 500px in width and 200px in height total. It has a red-ish background-color and
+        the margin on every side is 100px</div>
+    <div class="box-03-invisible">You should not be able to see this text at all</div>
+    <div class="box-04">This box is 100% in width and 150px in height total. It has a blue-ish background-color,
+        padding: 50px and
+        a yellow-ish 10px wide border. Pay attention to the width of this box, it should not be wider than the page
+        itself!</div>
+    <div class="box-05">You should not be able to see this text as well but there is a gap</div>
+    <div class="box-06">This box is 200x500, its background color is green-ish and its opacity is 0.5</div>
+    <div class="out-box">
+        <p>This box is 400x300 and its position is relative, its background color is light blue</p>
+        <div class="in-box">This box is 50x50 and its position is absolute inside its parent: 50px from the top
+            and 10px from the right. Its background color is pink</div>
+    </div>
+    <p>This button is 300px wide and 100px high and whenever I click on it, its text's
+        color turns blue</p>
+    <button class="button-01">Button 01.</button>
+    <p>This button is 200px wide and 50px high and whenever hover my mouse on it, its
+        background color turns brown</p>
+    <button class="button-02">Button 02</button>
+    <div class="box-07">This box is 200x500, its background color is blue-ish and whenever I hover my mouse over
+        it, the cursor turns in to a hand. There is a 200px space above this box.</div>
+</body>
+
+</html>
```

Comment:
úkol musí být odevzdán přes git commit + git push z tvé branche - nesmí se nahrát přes web github. Odevzdej to prosím opravený znova z IDE nebo z command line